docs/nipoppy/overview.md

```diff
@@ -1,39 +1,90 @@
+
+## Who is Nipoppy for?
+
+Anyone who wants to process datasets with imaging data and/or use datasets processed with Nipoppy.
+
+**Data managers**
+
+* People who process datasets, either for specific analyses or to share with others
+* Example use cases:
+    * Data curation
+        * Download/organize raw DICOM files
+        * Convert raw data to a BIDS directory structure
+        * Organize clinical data
+    * Data processing
+        * Run pre-existing processing pipelines
+        * Add scripts to run custom pipelines
+        * Do cross-dataset processing: running the same pipelines/versions on different datasets so that the outputs can be used together
+    * Data tracking
+        * Check processing failures and relaunch processing
+        * Generate bagel files for Neurobagel
 
+**Data users**
 
-## Objectives
-1. Standardized data i.e. convert DICOMs into BIDS
-2. Run commonly used image processing pipelines e.g. FreeSurfer, fMRIPrep
-3. Organize processed MR data inside `derivatives` directory
-4. Organize demographic and clinical assessment data inside `tabular` directory
-5. Run tracker scripts to populate `bagel.csv` with tabular and processing pipeline metadata
-6. Provide metadata to `NeuroBagel` to allow dashboarding and querying participants across multiple studies
-    
+* People who use tabular, derivative or other files produced by Nipoppy (e.g., from a shared Nipoppy-compliant dataset)
+* Example use cases:
+    * Data tracking
+        * Check availability of raw and/or derived data
+        * Check which pipelines and version have been run
+    * Data analysis
+        * Extract imaging features for downstream analyses from specific pipelines/versions
 
-## Organization
+## How does Nipoppy work?
+
+### Modules
+
+1. `Code`: Codebase [repo](code_org.md) for running and tracking workflows
+    * The codebase should start from the [Nipoppy template repository](https://github.com/neurodatascience/nipoppy). Additional custom scripts need to be added to process specific datasets.
+2. `Data`: [Dataset](data_org.md) organized in a specific directory structure
+    * This contains the data only and should not contain any code
+3. `Containers`: [Singularity/Apptainer](https://apptainer.org/) containers encapsulating processing pipelines
+
+### Organization
 Organization of `Code`, `Data`, and `Container` modules
 
-![mr_proc_org](../imgs/mr_proc_org.jpg)
+![nipoppy_org](../imgs/nipoppy_org.jpg)
+
+### Steps
+The Nipoppy workflow steps and linked identifiers (i.e. `participant_id`, `dicom_id`, `bids_id`) are shown below:
 
+![steps](../imgs/steps.jpg)
 
-## Steps
-The Nipoppy workflow steps and linked identifiers (i.e. participant_id, dicom_id, bids_id) are show below:
+## FAQ
```

Comment:
Question /  Answers to add
- Do I need to install anything?
- Can I use Nipoppy on a cluster?
- How do I use the dashboard?